biodatasets/nagel/nagel.py

```diff
@@ -0,0 +1,312 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from pathlib import Path
+from typing import List
+
+
+import datasets
+import pandas as pd
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{,
+  author    = {Kevin Nagel},
+  title     = {Automatic functional annotation of predicted active sites: combining PDB and literature mining},
+  journal   = {},
+  volume    = {},
+  year      = {2009},
+  url       = {https://www.ebi.ac.uk/sites/ebi.ac.uk/files/shared/documents/phdtheses/kevin_nagel.pdf},
+  doi       = {},
+  biburl    = {},
+  bibsource = {}
+}
```

Comment:
This bibtex might be more suitable: 
```suggestion
@phdthesis{nagel2009automatic,
  title={Automatic functional annotation of predicted active sites: Combining PDB and literature mining},
  author={Nagel, Kevin},
  year={2009},
  school={University of Cambridge}
  url= {https://www.ebi.ac.uk/sites/ebi.ac.uk/files/shared/documents/phdtheses/kevin_nagel.pdf},
}
```